packages/harness/package.json

```diff
@@ -10,7 +10,7 @@
 		"./package.json": "./package.json"
 	},
 	"peerDependencies": {
-		"@angular/cdk": "^13.3.2 || ^14.0.0-rc.0 || ^15.0.0-rc.0 || ^16.0.0-rc.0 || ^17.0.0"
```

Comment:
I believe I already told @jVerhaert via chat, but I don't want to use a `>= X` range. It's impossible to predict forward compatibility with `@angular/cdk`, so this range would be a lie.
(See also the `>= 8.43.0` eslint range issue we had earlier this year when eslint 9 came out)